packages/now-next/src/index.ts

```diff
@@ -368,18 +385,21 @@ export const build = async ({
   console.log('Installing dependencies...');
   await runNpmInstall(entryPath, ['--prefer-offline'], spawnOpts, meta);
 
-  let realNextVersion: string | undefined;
-  try {
-    realNextVersion = require(resolveFrom(entryPath, 'next/package.json'))
-      .version;
+  // Refetch Next version now that dependencies are installed.
+  // This will now resolve the actual installed Next version,
+  // even if Next isn't in the project package.json
+  nextVersion = await getNextVersion(entryPath);
```

Comment:
I think this is fine, but I want the Next.js team to look at it too

cc @ijjk @Timer

Reply:
Yeah after discussion this change seems fine since we still throw the error when we fail to detect the version after installing dependencies, [cont.](https://github.com/vercel/vercel/pull/4465#pullrequestreview-436006537)